### DISCUSSION TITLE
关于daily speech excercises语音的问题

### DISCUSSION THREAD

**Discussion by sesars** [ZuodaoTech/everyone-can-use-english#1086] [Q&A] [answered]
请问教程中的 daily speech excercises中的语音是用azure还是open ai生成的呢？
我看已经内置了语音和转好的文本

**Comment by an-lee** [comment, marked as answer]
open ai 生成的

**Comment by bluechallenge** [comment]
<每日练习语音生成> 相关的Python代码请参考这里，调用的是 OpenAI API：

https://1000h.org/sounds-of-american-english/8.4-daily-speech-exercises.html
